validator/specs/lib/readTilesetSpec.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+var readTileset = require('../../lib/readTileset');
+
+describe('readTileset', function() {
+    it('reads a tileset', function(done) {
+        expect(readTileset('./specs/data/Tileset/tileset.json')
+            .then(function(tilesetData) {
+                return (tilesetData['asset'] !== undefined)
```

Comment:
Use `defined` instead for this and below. Example: https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/blob/master/validator/specs/matchers/expectPromise.js